mib-generator/cw_mib_generator.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+
+# Copyright (C) Metaswitch Networks 2017
+# If license terms are provided to you in a COPYING file in the root directory
+# of the source code repository by which you are accessing this code, then
+# the license outlined in that COPYING file applies to your use.
+# Otherwise no rights are granted except for those provided to you by
+# Metaswitch Networks in a separate written agreement.
+
+"""Metaswitch Clearwater MIB Generator
+Usage:
+  cw_mib_generator.py [--cwc-mib-dir=DIR]
+
+Options:
+  --cwc-mib-dir=DIR  Directory containing Clearwater Core MIB fragment
+"""
+
+import re
+import sys
+import os.path
+from string import Template
+from docopt import docopt
```

Comment:
The import order isn't quite as spec'd. I think `autopep8` will fix that up for you. I'd suggest running that over all your code so you don't have to bother with PEP8 formatting yourself.